server/routes/incidents_sse.py

```
@@ -1,32 +1,35 @@
+from utils.cache.redis_client import get_redis_ssl_kwargs
 
 logger = logging.getLogger(__name__)
 
 incidents_sse_bp = Blueprint('incidents_sse', __name__)
 
-# Store active SSE connection queues per org: {org_id: [queue1, queue2, ...]}
-_active_connection_queues_by_org = {}
+_CHANNEL_PREFIX = "incidents:sse:"
 
 
 def broadcast_incident_update_to_user_connections(user_id: str, incident_data: dict, org_id: str = None):
```

Ruff (0.15.12)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)